src/borg/archiver.py

```diff
@@ -2644,11 +2644,13 @@ def define_exclude_and_patterns(add_option, *, tag_files=False, strip_components
                        type=parse_exclude_pattern, action='append',
                        help='exclude paths matching PATTERN')
             add_option('--exclude-from', metavar='EXCLUDEFILE', action=ArgparseExcludeFileAction,
-                       help='read exclude patterns from EXCLUDEFILE, one per line')
+                       help='read exclude patterns from EXCLUDEFILE, one per'
+                       'line. These entries are processed by borg and not the shell')
             add_option('--pattern', metavar='PATTERN', action=ArgparsePatternAction,
                        help='experimental: include/exclude paths matching PATTERN')
             add_option('--patterns-from', metavar='PATTERNFILE', action=ArgparsePatternFileAction,
-                       help='experimental: read include/exclude patterns from PATTERNFILE, one per line')
+                       help='experimental: read include/exclude patterns from PATTERNFILE, one per line'
+                       'These entries are processed by borg and not the shell')
```

Comment:
maybe this is a) too long fo here and b) not really helpful as the user would need to be already aware of what the specialities of shell processing are.

so guess we better add this to the (long) docs and also give an example there that shows that spaces are not escaped in pattern files.

Reply:
Sure I'll do this now.